packages/caliper-cli/lib/bind/config.yaml

```diff
@@ -81,3 +81,6 @@ sut:
             settings:
             - *new-node-old-grpc
         latest: *composer-latest
+    ethereum:
+        1.2.1: &ethereum-latest
+            packages: ['web3@1.2.1']
```

Comment:
After this line, add the "latest" binding (same level as 1.2.1), and reference the `ethereum-latest` anchor, which will "copy-paste" the packages section
```
latest: *ethereum-latest
```